molgenis-data-annotators/src/test/java/org/molgenis/data/annotation/resources/impl/ResourceImplTest.java

```diff
@@ -35,7 +35,7 @@ public void beforeMethod()
 			@Override
 			public RepositoryFactory getRepositoryFactory()
 			{
-				return new TabixVcfRepositoryFactory("cadd");
+				return null;//new TabixVcfRepositoryFactory("cadd");FIXME
```

Comment:
If this is not supposed to be fixed right now, add to meta data editing to do doc